### PR TITLE
Fix flake8 errors and add linter tests

### PR DESCRIPTION
**Pull request by cottsay**

### CODE REVIEW

File: test/test_flake8.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
```
Comment (wjwwood) [inline test/test_flake8.py:1]: If we're copying this from somewhere, do we keep the C year? If not, could we just make it BSD (given how trivial this is)?

Comment (cottsay) [reply test/test_flake8.py:1]: Fair point. I grabbed these from `rqt_reconfigure`, so I'll revert to the original year, 2019.

File: src/rqt_topic/topic_widget.py [hunk 2]
```diff
@@ -250,7 +250,7 @@ def _update_topics_data(self):
 
             else:
                 rate_text = ''
-                bytes_per_s = None
+                # bytes_per_s = None
```
[the remark this answers is not in the document]
Comment (cottsay) [reply src/rqt_topic/topic_widget.py:253]: It's part of a larger block that's commented out. If you expand the diff a little on either end, you can see there are quite a few other lines. I really don't have much context on the state of this package yet, so I'd rather not drop it.

File: test/test_flake8.py (hunk 1 above)
Comment (cottsay) [reply test/test_flake8.py]: 98fa4fc

File: src/rqt_topic/topic_widget.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (wjwwood) [reply src/rqt_topic/topic_widget.py:253]: Ok 👍

**Review by wjwwood — APPROVED** [review]

**Pull request merged**